.circleci/config.yml

```diff
@@ -55,7 +55,7 @@ workflows:
       - test:
           matrix:
             parameters:
-              python_version: ["2.7", "3.6", "3.7", "3.8"]
```

Comment:
Unfortunately we still have a [2.7 project](https://github.com/closeio/sync-engine/blob/798fe440e3af73454943f3ba2b28c69814588599/requirements_frozen.txt#L60) using this. So ideally we would keep 2.7 around for a bit longer. If it becomes a hinderance we can just try to keep that project on the old version of limitlion and worse case do a branch to fix any issues until we can get sync-engine to py3.

Reply:
The issue is freezefrog and this `encoding` kwarg. It can be probably easily fixed, but there might be more issues. 
https://app.circleci.com/pipelines/github/closeio/limitlion/43/workflows/4039a419-fa4d-4c1c-add6-cc6ffd84cf33/jobs/314
https://github.com/closeio/freezefrog/blob/master/setup.py